### ISSUE TITLE
Missing config.reset_password_mailer when running from master

### ISSUE THREAD

**Issue by bzf** [Sorcery/sorcery#137]
When I'm trying to run our application using the `master` branch rather than the version (0.12.0) from Rubygems our app crashes with the error:

```
To use reset_password submodule, you must define a mailer (config.reset_password_mailer = YourMailerClass).
```

I've bisected it between `0.12.0..master` and landed on this [this commit][1].

Please ask if there's anything you need to know or if there's anything I could do to help figure this out. 😃 

_EDIT_:
We're using Ruby 2.4.4 and Rails 5.0.7.

[1]: https://github.com/Sorcery/sorcery/commit/2fea1d78361976286c79795d0740b89f8e4156b6

**Comment by athix** [comment]
@Ch4s3 we're sending everything into the module instead of the `Base` class that `ApplicationController` typically inherits. We'll want to update that code to instead inject into `ActionController::Base` _**and**_ `ActionController::API` individually, depending on their load status (or just send regardless if it doesn't force unnecessary loading of the `Base` class).

**Comment by denniscastro** [comment]
Reopen it!. 

In controllers is all ok, the helpers are injected, but the model doesnt it and get the same error.

``
    ArgumentError: To use reset_password submodule, you must define a mailer (config.reset_password_mailer = YourMailerClass)
``

**Comment by denniscastro** [comment]
Hi.

It is fixed now, with the edge version

**Comment by athix** [comment]
Ah, yeah this is the issue I fixed on #159. Closing this out again.
